### ISSUE TITLE
Changing keybinds cause freezes after launch

### ISSUE THREAD

**Issue by Mrcubix** [ppy/osu#26836]
### Type

Performance

### Bug description

Changing any of the keybinds right after launching osu!lazer cause it to freeze for a short amount of time.

### Screenshots or videos

https://github.com/ppy/osu/assets/39861216/ee204430-1c22-4ab1-a249-78205b79c38b

### Version

2024.130.2

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14103763/compressed-logs.zip)

**Comment by Kiciuk** [comment]
Doesn't happen for me, not even slight spike in frametime.
Maybe background calculation is doing something?

> 2024-01-30 20:17:52 [verbose]: Upgrading scores to new scoring algorithm (200 of 6644)
> 2024-01-30 20:17:58 [verbose]: Scrolling to RulesetBindingsSection..
> 2024-01-30 20:17:59 [verbose]: Finished scrolling to RulesetBindingsSection!

IDK i don't know much thats my guess.

**Comment by bdach** [comment]
Yes it's likely that. I swear there was an issue thread for this already but can't find it now.

**Comment by Mrcubix** [comment]
Here is one more sample w/ video, no import is happening in the background as far as i'm aware this time

https://github.com/ppy/osu/assets/39861216/77bfbf89-a6e1-41fd-8335-a6fb52df626f

[compressed-logs.zip](https://github.com/ppy/osu/files/14104143/compressed-logs.zip)

**Comment by Jafeth12** [comment]
This also happens to me.

**Comment by frenzibyte** [comment]
> This also happens to me.

Thank you for your feedback, but please avoid posting comments like this since they're not adding anything to this thread nor is there anyone asking whether anyone else can reproduce this issue.

**Comment by Alurall** [comment]
For me it freezes for over 60s (sometimes up to 3-4mins) each time I change a keybind. This happens always, right after launch or some after. Only thing I can add is that my Osu! runs on a HDD and it does seem to impact the duration of the freeze.

**Comment by Mrcubix** [comment]
The issue is present without background processing, but background processing seems to amplify this issue.

With the score multiplier background processing atm, changing keybinds locks the game up for multiple minutes.

Though for this case, I noticed it's only the graphic thread that is locked up, if i move the cursor, then i can hear the SFX from hovering on some of the UI Controls.

Edit : I wanted to get the logs for this one, but after attempting to reproduce the issue again, my game locked up, pàreventing me from exporting logs (Why does it seem to only be possible from inside the game settings in the first place, what happens when the game just crashes, are logs still expected from somewhere ? This needs some extra explanation in the bug report template)

Edit II : Found some logs, i don't know if it's any different from what the game is supposed to "export", and i don't want to spend another day looking into the code just get get spit in the face, anyway, here are what i've found : 

[logs.zip](https://github.com/user-attachments/files/29750372/logs.zip)
